sklearn/manifold/t_sne.py

```diff
@@ -808,17 +812,27 @@ def _tsne(self, P, degrees_of_freedom, n_samples, random_state,
         # * early exaggeration with momentum 0.5
         # * early exaggeration with momentum 0.8
         # * final optimization with momentum 0.8
+
+        # TODO sklearn 0.20: remove this warning
+        if self.learning_rate == "200 (new default)":
```

Comment:
I don't know exactly what to do here. Should I set self.learning_rate in fit? That means self.learning_rate is different from what we set in the constructor. That is not a good practice. I would like to keep it here to set a local variable `learning_rate` or pass the learning_rate explicitly to `_tsne` as a parameter.